cmd/build/command.go

```diff
@@ -129,3 +150,38 @@ func validateDockerfile(file string) error {
 	_, _, err = instructions.Parse(parsedDockerfile.AST)
 	return err
 }
+
+func (*Command) loadContext(ctx context.Context, options *types.BuildOptions) error {
+	ctxOpts := &contextCMD.ContextOptions{}
```

Comment:
The implementation of this function is slightly different between both versions (v1 and v2) and it seems we just got the v1 implementation, not sure this is the right thing to do, we might be breaking some cases. We need to review it carefully

Reply:
already talked about that with @ifbyol. if you're curious, take a look at how the function looked like after that.